src/app/app.server.ts

```diff
@@ -0,0 +1,29 @@
+import express, { Application } from 'express'
+import { Logger } from '../utils/logger';
+
+class App {
```

Comment:
Good approach. I like it;

Just like a comment, the OOP is a good idea when you are trying to encapsulate some functionality, but the question is, how many times you will need to create a new Express Application? 

The classes will be useful to create routers and request handlers. 👍

Reply:
@nestorvanz Yes I agree, maybe a singleton pattern in this case could be useful?
 I just did this kind of a proof of concept with a class and definitely there is more room for improvement! 

Appreciate your comment. 👍

Reply:
You will have the opportunity to prove your OOP skills in the next sessions 😝